notebooks/ray-examples/ray-tune.ipynb

```diff
@@ -0,0 +1,1375 @@
+{
```

Comment:
This notebook also shows using ray with sklearn which seems to be a good use case. Would renaming the notebook or adding markdown to highlight that make sense?